### PR TITLE
rm chart value 'allowNamespaceDiscovery' fixing issue #5422

### PR DESCRIPTION
**Pull request by giulianopz**
Fix issue #5422.

### CODE REVIEW

**Comment by antgamdia** [comment]
Thanks for your contribution, @giulianopz . Please, have a look at this project we are using to auto-generate the readme: https://github.com/bitnami-labs/readme-generator-for-helm. Then, run it and update the README.md with the change you performed in the values.yaml file.
Also, please bump up the chart version to sth like, `x.x.x-devN+1`, for instance, `11.0.1-dev2`

File: chart/kubeapps/README.md [hunk 1]
```diff
@@ -696,19 +695,61 @@ kubectl delete namespace kubeapps
...
+    - [Configuring Initial Repositories](#configuring-initial-repositories)
+    - [Enabling Operators](#enabling-operators)
+    - [Exposing Externally](#exposing-externally)
+      - [LoadBalancer Service](#loadbalancer-service)
+      - [Ingress](#ingress)
+        - [Hosts](#hosts)
+        - [Annotations](#annotations)
+        - [TLS](#tls)
+  - [Upgrading Kubeapps](#upgrading-kubeapps)
+  - [Uninstalling the Chart](#uninstalling-the-chart)
```
Comment (antgamdia) [inline chart/kubeapps/README.md:728]: I don't think we need to add these changes, we only want to keep the FAQ section, not the whole table of contents of the readme at this point. Please, revert these changes

File: chart/kubeapps/values.schema.json [hunk 2]
```diff
@@ -1,125 +1,2886 @@
 {
-  "$schema": "http://json-schema.org/schema#",
```
Comment (antgamdia) [inline chart/kubeapps/values.schema.json:2]: Thanks for using the tool to also generate the underlying JSON schema. That said, currently we don't want to use this schema. Just for you to know, in the latest [v2.6.0 release](https://github.com/vmware-tanzu/kubeapps/releases/tag/v2.6.0), we migrated to full json schema based form. Along with this chance, the [Bitnami Application Catalog](https://github.com/bitnami/charts/) is gonna autogenerate the schemas using this tool soon. Hence, we want to switch the schema once they also perform the change. Hope it makes sense for you.

In short, please revert the changes in this file.

**Review by antgamdia — CHANGES_REQUESTED** [review]
Thanks for using the tool I suggested to autogenerate the readme file. Find below just a couple of minor changes before merging the PR (mainly removing some unwanted changes). Apart from that, +1.

Also, a github action check is currently failing; we are working to fix it, it's not your PR's fault :P

**Comment by giulianopz** [comment]
> Thanks for using the tool I suggested to autogenerate the readme file. Find below just a couple of minor changes before merging the PR (mainly removing some unwanted changes). Apart from that, +1.
> 
> Also, a github action check is currently failing; we are working to fix it, it's not your PR's fault :P

Unwanted changes reverted as requested.
Actually, thanks for suggesting that tool. I didn't know it: very useful.

**Review by antgamdia — APPROVED** [review]
Great, thanks for your contribution :)

**Comment by antgamdia** [comment]
@giulianopz , there's an additional pending thing. According to [our contributing guidelines](https://github.com/vmware-tanzu/kubeapps/blob/main/CONTRIBUTING.md), all the commits must be signed.
That this is twofold: 1) sign them with GPG [1]; 2) add the DCO in each commit's msg [2]. You might need to force push the changes to your branch in order to replace the non-signed commits.

If you face any problems, happy to assist via slack. Let's try to get it to merge before the end of October, so that it can count for the hacktoberfest!  

[1] https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits
[2] https://wiki.linuxfoundation.org/dco

**Comment by giulianopz** [comment]
> @giulianopz , there's an additional pending thing. According to [our contributing guidelines](https://github.com/vmware-tanzu/kubeapps/blob/main/CONTRIBUTING.md), all the commits must be signed. That this is twofold: 1) sign them with GPG [1]; 2) add the DCO in each commit's msg [2]. You might need to force push the changes to your branch in order to replace the non-signed commits.
> 
> If you face any problems, happy to assist via slack. Let's try to get it to merge before the end of October, so that it can count for the hacktoberfest!
> 
> [1] https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits [2] https://wiki.linuxfoundation.org/dco

Never done before. Happy that contributing to a simple issue taught me a couple of things. Thanks!

**Comment by antgamdia** [comment]
Thank you! LGTM, merging straight away :)
